apps/admin-gui/src/app/vos/components/application-detail/application-detail.component.ts

```diff
@@ -99,7 +99,7 @@ export class ApplicationDetailComponent implements OnInit {
   rejectApplication() {
     const config = getDefaultDialogConfig();
     config.width = '500px';
-    config.data = {applicationId: this.application.id};
+    config.data = {applicationId: this.application.id, theme: 'vo-theme'};
```

Comment:
This component is not only used in Vo, it is also used in member section and group section.